crates/ruff_python_parser/src/parser.rs

```diff
@@ -714,9 +715,9 @@ except OSError as e:
     raise ExceptionGroup("eg",
         [ValueError(1), TypeError(2), OSError(3), OSError(4)])
 except* TypeError as e:
-    print(f'caught {type(e)} with nested {e.exceptions}')
+    print('caught', type(e), 'with nested', e.exceptions)
 except* OSError as e:
-    print(f'caught {type(e)} with nested {e.exceptions}')"#,
+    print('caught', type(e), 'with nested', e.exceptions)"#,
```

Comment:
The tests are for `try ... except ...` so let's use normal strings to avoid putting the test behind feature flag.